librz/include/rz_asm.h

```diff
@@ -124,6 +125,10 @@ typedef struct rz_asm_t {
 	HtPP *flags;
 	int seggrn;
 	bool pseudo;
+	RzPVector *pvec;
```

Comment:
Please check how it's done in https://github.com/rizinorg/rizin/commit/a592ea1e8c503e663b37e58c4beccecb40d2cb5e

See also this:
```
Missing type comment at <rizin/librz/include/rz_asm.h:128:13> (token is not a comment)
Error: Process completed with exit code 1.
```

Reply:
Okay I will